### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by Rupakpoddar**
The two deleted libraries have been removed intentionally. They have now been combined into this better version of the library that is more reliable and supports more devices: https://github.com/Rupakpoddar/FirebaseArduino

All three libraries created and managed by the same developer (me).

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @Rupakpoddar. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by per1234 — COMMENTED** [review]
Hello again @Rupakpoddar. In the future, if you want to request any additional removals, etc., please do that by submitting an additional pull request.

I happened to notice that you updated this pull request to remove two more URLs, so you don't need to submit another pull request for these additional two removal requests. However, I could have easily missed that (this repository generates a HUGE number of notifications and it is difficult for me to carefully check every one of them), so make sure to always open new pull requests for any future operations you want to perform here.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/09/12 15:57:21 Loaded 7296 libraries from DB
Removing ESP32 Firebase
Success!
```
```
2024/09/12 15:58:23 Loaded 7295 libraries from DB
Removing ESP8266 Firebase
Success!
```
```
2024/09/12 15:59:36 Loaded 7294 libraries from DB
Removing ESP32 Webhook
Success!
```
```
2024/09/12 16:00:29 Loaded 7293 libraries from DB
Removing ESP8266 Webhook
Success!
```

**Pull request merged**
